hypervisor/arch/x86/trusty.c

```diff
@@ -63,12 +63,6 @@ static void create_secure_world_ept(struct acrn_vm *vm, uint64_t gpa_orig,
 	void *sub_table_addr, *pml4_base;
 	uint16_t i;
 
-	if ((vm->sworld_control.flag.supported == 0UL)
-			|| (vm->arch_vm.sworld_eptp != NULL)) {
-		pr_err("Sworld is not supported or Sworld eptp is not NULL");
-		return;
-	}
-
```

Comment:
Why we delete this one? Assumption changes?

Reply:
Just move it to initialize_trusty. create_secure_world_ept also be called in restore_sworld_context where these condictions are certainly met.